docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -194,9 +184,21 @@ Assume a simple set-up with 2 priority levels, P=1 100% healthy.
 | 0%                         | 0%             |   100%          |
 +----------------------------+----------------+-----------------+
 
-If P=1 becomes unhealthy, it will continue to take spilled load from P=0 until the sum of the health
-P=0 + P=1 goes below 100. At this point the healths will be scaled up to an "effective" health of
-100%.
+.. attention::
+
+  In order for the load distribution algorithm and normalized total health calculation to work
+  properly, the number of hosts in each priority level should be close. Envoy assumes that for
```

Comment:
I'd probably spell out the exact assumption made by Envoy (each priority is able to handle (100% * overprovision factor) of the traffic) instead of dictating that the number of hosts should be similar. Your example can still talk about a host count imbalance